### PR TITLE
Add Modules For Drag And Drop And Check Task Completition

### PR DESCRIPTION
**Pull request by JbirdL86**
Added 4 js files and imported them as modules. 

- eventHandler.js: It contains and assigns all the event listeners to all the items and includes the update function which updates the array to be saved on the local storage.
- drag.js: It contains the function that swaps the HTML from the current item container to the old item container.
- completed.js: It contains a function that updates the task completed value and sets a class to the input to mark it down.
- sorage.js: It contains a method to save the tasks array in the local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,27 +23,42 @@ const tasksList = [
...
+  li.classList.add('task-item');
 
   li.innerHTML = `
-    <li class="task-item">
-      <label class="task-label">
-        <input type="checkbox" value="${task.completed}">
-        <p class="task-description">${task.description}</p>
-      </label>
-    <i class="fas fa-ellipsis-v"></i>
-    </li>`;
```
Comment (abruzy) [inline src/index.js:33]: - [ ] Kindly fix the add functionality for your app, when I enter a task and press the enter key, it doesn't add it. Please look into this and fix it

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @JbirdL86 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_1. Let's make your GitHub looks more professional by adding instructions on how to get a local copy up and running._
_2. Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (JbirdL86) [reply src/index.js:33]: >     * [ ]  Kindly fix the add functionality for your app, when I enter a task and press the enter key, it doesn't add it. Please look into this and fix it

Thanks for your review. However, I think that's part of the next exercise requirements. The requirements for this task are available here https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md. Please let me know if it's neccesary.

**Review by dasileker — APPROVED** [review]
### **Hello @JbirdL86** 
### **Project Approved✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it  
Congratulations! 🎉 

###  Optional

- Please next time add the Vscode folder to the **`.gitignore`** file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**